src/main/java/org/traccar/protocol/RuptelaProtocolDecoder.java

```diff
@@ -151,9 +176,19 @@ private void decodeParameter(Position position, int id, ByteBuf buf, int length)
                     position.set(Position.KEY_ALARM, Position.ALARM_ACCIDENT);
                 }
                 break;
+            case 415:
+                if (readValue(buf, length, false) != 1) {
```

Comment:
What about this one?

Reply:
Why would unknown be an alarm?

Reply:
Hm, I was thinking to err on the side of caution (eg. everything that is not connected, is setting an alarm), but don't have a better reason than this. But don't have a strong opinion on this, so we can change it no problems.

Reply:
Let's only use 0 as antenna cut. Im actually not entirely sure this is the right thing to use here. Some devices report it as a separate alarm, but in this case we're kind of using the state.

Reply:
Ok, changed it and will revisit this later, when I take a look at the events and what exactly does Ruptela send when it detects that the GPS antenna is disconnected.